equinox/nn/normalization.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Optional, Tuple, Union
+
+import jax
+import jax.numpy as jnp
+
+from ..custom_types import Array
+from ..module import Module, static_field
+
+
+_shape_t = Union[int, Tuple[int], List[int]]
```

Comment:
`Union[int, Sequence[int]]` is probably what's meant here. (c.f. the implementation of something similar for the convolutional layers.) And as that's a bit shorter I think `_shape_t` can be inlined.